### PR TITLE
Implement fetching of keys from un-registered cryptonym DIDs.

### PR DESCRIPTION
**Pull request by dmitrizagidulin**
Enables PR in another project.

### CODE REVIEW

File: tests/VeresOne.spec.js [hunk 1]
```diff
@@ -70,6 +70,65 @@ describe('methods/veres-one', () => {
...
+        .reply(404);
+
+      _nockLedgerAgentStatus();
+
+      // eslint-disable-next-line max-len
+      const unregisteredKey = 'did:v1:test:nym:z6MkesAjEQrikUeuh6K496DDVm6d1DUzMMGQtFHuRFM1fkgt#z6MkesAjEQrikUeuh6K496DDVm6d1DUzMMGQtFHuRFM1fkgt';
+      const result = await v1.get({did: unregisteredKey});
+
+      expect(result.doc).to.eql({
+        '@context': [
```
Comment (dmitrizagidulin) [inline tests/VeresOne.spec.js:91]: Is it correct / acceptable, to use the DID contexts with a returned key object?

Comment (dlongley) [reply tests/VeresOne.spec.js:91]: Well, let's say yes *for now*. That might need to change in the future based on how the contexts get scoped when they are updated to JSON-LD 1.1. But being able to include the contexts with the keys is an important consideration for that process when we get there. I think it's fine for now.

File: tests/VeresOne.spec.js [hunk 2]
```diff
@@ -70,6 +70,65 @@ describe('methods/veres-one', () => {
...
+        LEDGER_AGENTS_DOC;
+      nock(ledgerQueryService)
+        .post('/?id=' + encodeURIComponent(UNREGISTERED_NYM))
+        .reply(404);
+
+      _nockLedgerAgentStatus();
+
+      // eslint-disable-next-line max-len
+      const unregisteredKey = 'did:v1:test:nym:z6MkesAjEQrikUeuh6K496DDVm6d1DUzMMGQtFHuRFM1fkgt#z6MkesAjEQrikUeuh6K496DDVm6d1DUzMMGQtFHuRFM1fkgt';
+      const result = await v1.get({did: unregisteredKey});
```
Comment (dlongley) [inline tests/VeresOne.spec.js:88]: @dmitrizagidulin The one thing I don't like about this API is that the value you're passing for `did` here is *not* a DID. That's why I think `url` or something should be the parameter name ... and if `did` is used, it should probably throw an error if it's not a DID. But that can be addressed in the future.

Comment (dmitrizagidulin) [reply tests/VeresOne.spec.js:88]: I agree. I think I went so far as put it an alias, so you could either say `.get({did})` or `.get({url})` for this very reason, but now I can't find it - I think it was at another library. But, agreed.

**Pull request merged**